src/azure-cli/install.sh

```diff
@@ -197,6 +198,28 @@ if [ ${#AZ_EXTENSIONS[@]} -gt 0 ]; then
     done
 fi
 
+if [ "${AZ_INSTALLBICEP}" = "true" ]; then
+    # Properly install Azure Bicep based on current architecture
+    # The `az bicep install` command installs the linux-x64 binary even on arm64 devcontainers
+    # The `az bicep install --target-platform` could be a solution; however, linux-arm64 is not an allowed value for this argument yet
+    # Manually installing Bicep and moving to the appropriate directory where az expects it to be
+    apt-get install curl -y
```

Comment:
Perhaps add `--no-install-recommends` for speed?

```suggestion
    apt-get install curl -y --no-install-recommends
```